pset-functions.js

```diff
@@ -76,6 +143,54 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+// declare a word
+let word = "cabbage";
+
+// convert the word to uppercase becaues the keys in the alphabet object are capital letters
+word = word.toUpperCase();
+
+// creates an object with the scores for every letter. Each letter in the alphabet is a key, and the corresponding value is that letter's score
+const alphabet = {
+    A: 1,
+    B: 3,
+    C: 3,
+    D: 2,
+    E: 1,
+    F: 4,
+    G: 2,
+    H: 4,
+    I: 1,
+    J: 8,
+    K: 5,
+    L: 1,
+    M: 3,
+    N: 1,
+    O: 1,
+    P: 3,
+    Q: 10,
+    R: 1,
+    S: 1,
+    T: 1,
+    U: 1,
+    V: 4,
+    W: 4,
+    X: 8,
+    Y: 4,
+    Z: 10
+}
+
+//declares three variables to be used in a for loop to loop through the word. Letter holds the current letter, i is the index in the loop, and sum is the running tally of the word's score as each letter is added.
+let letter, i, sum = 0;
+
+// loops through each letter in the word, finds the value associated with the letter from the alphabet object, and adds it to the sum (score)
+for (i = 0; i < word.length; i++) {
+    letter = word[i];
```

Comment:
Really great use of a for loop coupled with the understanding on how object key/value pairs work! Be a little careful with declaring every variable in the global scoop (outside function block for example); it can potentially interfere with code throughout your file.